packages/office-ui-fabric-react/src/components/ActivityItem/examples/ActivityItem.Example.scss

```diff
@@ -0,0 +1,9 @@
+:global {
+  .ms-activityItem-exampleRoot {
+    margin-top: 20px;
```

Comment:
why use global here?

Reply:
These are globals specifically for the example, I didn't want to apply margin to the root and make users fight against it. The styles here don't get applied if they aren't global.

@dzearing, Is there a better way I should be doing this?